docs_src/microservices/support/scheduler/Ch-Scheduler.md

```diff
@@ -0,0 +1,110 @@
+# Scheduler
+
+![image](EdgeX_SupportingServicesScheduling.png)
+
+## Introduction
+
+!!! edgey "Edgex 2.0"
```

Comment:
There isn't any content for this edgey note.

Should look something like this (note the 4 space indent):
```
!!! edgey "Edgex 2.0"
    For EdgeX 2.0 xyz is new and the REST API provided by the Support Scheduler has changed to use DTOs (Data Transfer Objects) for all responses and for all POST/PUT/PATCH requests. All query APIs (GET) which return multiple objects, such as /all, provide `offset` and `limit` query parameters. 
```

Reply:
Updated. There is no new API in the Support Scheduler, so only removed 'xyz is new'.